EventFilter/GEMRawToDigi/interface/GEBdata.h

```diff
@@ -65,20 +65,24 @@ namespace gem {
       m_Vwt = 0x0fff & (word >> 36);  /*!<VFAT word count*/
       m_InFu = 0x0f & (word >> 35);   /*!<InFIFO underflow*/
       m_Stuckd = 0x01 & (word >> 34); /*!<Stuck data*/
+      m_OHBC = 0x0fff & (word >> 20); /*!OH BC*/
```

Comment:
Please update the comment at L60 with the two new contents

Reply:
done